AppCenterAuth/AppCenterAuth/MSAuthErrors.h

```diff
@@ -19,7 +19,12 @@ static NSString *const kMSACAuthErrorDomain = MS_APP_CENTER_BASE_DOMAIN @"Auth.E
 /**
  * Error code for Auth.
  */
-NS_ENUM(NSInteger){MSACAuthErrorServiceDisabled = -420000, MSACAuthErrorPreviousSignInRequestInProgress = -420001,
-                   MSACAuthErrorSignInBackgroundOrNotConfigured = -420002, MSACAuthErrorSignInWhenNoConnection = -420003};
+NS_ENUM(NSInteger){// System framework errors
+                   MSACAuthErrorServiceDisabled = 101,
```

Comment:
Error codes defined by us do not have to be like `-42XXXX`, there's a bug opened to fix the same thing in MSData module.

But let's not use 3-digit like `1XX` or `2XX` which may confuse people with HTTP status codes.

My recommendation is to use 4-digit number like `1XXX` or `2XXX`.